static/usage/v7/select/customization/custom-toggle-icons/react.md

```diff
@@ -0,0 +1,24 @@
+```tsx
+import React from 'react';
+import { IonItem, IonList, IonSelect, IonSelectOption } from '@ionic/react';
+function Example() {
+  return (
+    <IonList>
+      <IonItem>
+        <IonSelect
+          interface="popover"
+          toggleIcon="add"
+          toggleIconWhenOpen="remove"
```

Comment:
Ionic React needs the icons imported from `ionicons/icons` and then passed to the props

```typescript
import { add, remove } from 'ionicons/icons';

...

toggleIcon={add}
```